python/lsst/sattle/logs/sattle.log

```diff
@@ -0,0 +1,22 @@
+2025-03-28 20:30:51,146 INFO main Server ready!
```

Comment:
Did you mean to include this logfile in the commit?

Reply:
Yes this needs to be included otherwise the logger doesn't work correctly.

Reply:
I don't understand why the logger needs a logfile with these lines in it to work... but I will take your word for it